app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/pref/Prefs.kt

```diff
@@ -29,10 +29,20 @@ object Prefs : KotprefModel() {
     var previousSessionTabId: Int by intPref(
             default = context.integer(R.integer.pref_default_value_previous_session_tab_id)
     )
+    var previousSessionScrollPosition: Int by intPref(
```

Comment:
Can you create PreviousSessionPrefs with this properties? 👀 

```
previousSessionTabMode
previousSessionTabId
previousSessionScrollPosition
previousSessionScrollOffset
```

Reply:
I want to confirm.
Do you mean that "Create **PreviousSessionPrefs.kt** file and Separate the parameter from Prefs.kt "?
Or "Create **object PreviousSessionPrefs into Prefs.kt**"?

In either case, I think that it will be more better than current implement.

Reply:
I want to separate! 🙋

Reply:
Thank you for answer! 🙇 
I try it.